distribution/lib/Standard/Table/0.0.0-dev/src/Data/Group_By.enso

```diff
@@ -0,0 +1,25 @@
+from Standard.Base import all
+
+## Group By Key
+type Group_By_Key
```

Comment:
This comment just repeats the name. IMO we shouldn't have comments like this, because they do not really add anything. I'm not really sure what to write here, but it just looks weird. Maybe in this case we can just have no comment at all... But maybe we can write some intention here, for example 'An internal key used for performing the group by operation in memory.' or something like that.